src/ert/job_runner/cli.py

```diff
@@ -62,16 +76,27 @@ def main(args):
     try:
         with open(JOBS_FILE, "r") as json_file:
             jobs_data = json.load(json_file)
+            experiment_id = jobs_data.get("experiment_id")
             ee_id = jobs_data.get("ee_id")
             ee_token = jobs_data.get("ee_token")
             ee_cert_path = jobs_data.get("ee_cert_path")
             evaluator_url = jobs_data.get("dispatch_url")
+            experiment_url = None
+            # uncomment once protobuf's ready
+            # see https://github.com/equinor/ert/issues/3437
+            experiment_url = jobs_data.get("experiment_url")
```

Comment:
Is the line supposed to be commented, or should we always set it? (hence the comment on line 85 would be wrong)

Reply:
`experiment_url` is gone now.